File: PSW-projekat/hospital-be-main/src/HospitalLibrary/Core/DTOs/BedDto.cs
using HospitalLibrary.Core.Model.Enums;

namespace HospitalLibrary.Core.DTOs
{
    public class BedDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public BedState BedState { get; set; }
        public virtual PatientDto PatientDto { get; set; }
        public int Quantity { get; set; }
        public BedDto() { }
        public BedDto(int id, string name, BedState bedState, PatientDto patientDto, int quantity)
        {
            Id = id;
            Name = name;
            BedState = bedState;
            PatientDto = patientDto;
            Quantity = quantity;
        }
    }
}

File: PSW-projekat/PZL-Angular/src/app/modules/hospital/model/bedDto.ts
import { BloodBankRegistrationComponent } from "../../blood-banks/blood-bank-registration/blood-bank-registration.component";
import { BedState } from "./bedState";
import { PatientDto } from "./patientDto";

export class BedDto {
    id: number = 0;
    deleted: boolean;
    name: string = '';
    bedState: BedState;
    patientDto: PatientDto;
    quantity: number = 0;

    public constructor(id: any, deleted: any, name: any, bedState:any, patientDto: any, quantity: any) {
        this.id = id;
        this.deleted = deleted;
        this.name = name;
        this.bedState = bedState;
        this.patientDto = patientDto;
        this.quantity = quantity;

    }
}